association-frontend/components/button/button.tsx

```diff
@@ -0,0 +1,56 @@
+import classnames from "classnames";
+import React from "react";
+
+type ButtonProps = {
+  disabled?: boolean;
+  loading?: boolean;
+  type?: "button" | "reset" | "submit";
+  fullWidth?: boolean;
+  onClick?: () => void;
+};
+
+const Button: React.FC<ButtonProps> = ({
+  type,
+  children,
+  fullWidth = false,
+  ...props
+}) => {
+  return (
+    <button
+      type={type}
+      // group relative w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500
+      className={classnames(
```

Comment:
so many classes 😨

Reply:
the button component is a bit messed up :D